addon/components/base/bs-form.js

```diff
@@ -220,26 +246,40 @@ export default Component.extend({
    * @private
    */
   submit(e) {
+    this.set('isSubmitting', true);
+
     if (e) {
       e.preventDefault();
     }
     let model = this.get('model');
 
     this.get('onBefore')(model);
```

Comment:
I wonder, should we also introduce promise support for this action?

Reply:
Yeah, but should be done in a separate PR shouldn't it?

Reply:
sure